src/main/java/com/thealgorithms/datastructures/heaps/LeonardoHeapHelper.java

```diff
@@ -0,0 +1,45 @@
+package com.thealgorithms.datastructures.heaps;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// TODO: CHeck file name. Check if the file needs to be here or in bitwose operations.
+public final class LeonardoHeapHelper {
+
+    private LeonardoHeapHelper() {
+    }
+
+    public static ArrayList<Integer> findConsecutiveTreeIndices(int num) {
```

Comment:
Better use List<Integer> here as It's best to return the most generic type that's appropriate for your interface.